File: src/main/java/me/kuku/yuq/pojo/BiliBiliPojo.java
package me.kuku.yuq.pojo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class BiliBiliPojo {
    private String userId;
    private String name;
    private String id;
    private String rid;
    private Integer type;
    private Long time;
    private String text;
    private String bvId;
    private List<String> picList = new ArrayList<>();
    private Boolean isForward;
    private String forwardUserId;
    private String forwardName;
    private String forwardId;
    private Long forwardTime;
    private String forwardText;
    private String forwardBvId;
    private List<String> forwardPicList = new ArrayList<>();
    public BiliBiliPojo(String userId, String name){
        this.userId = userId;
        this.name = name;
    }
}

File: src/main/kotlin/me/kuku/yuq/pojo/BiliBiliPojo.kt
package me.kuku.yuq.pojo

data class BiliBiliPojo(
        var userId: String = "",
        var name: String = "",
        var id: String = "",
        var rid: String = "",
        var time: Long = 0,
        var text: String = "",
        var bvId: String? = null,
        var picList: MutableList<String> = mutableListOf(),
        var isForward: Boolean = false,
        var forwardUserId: String? = null,
        var forwardName: String? = null,
        var forwardId: String? = null,
        var forwardTime: Long? = null,
        var forwardText: String? = null,
        var forwardBvId: String? = null,
        var forwardPicList: MutableList<String> = mutableListOf()
)